frontend/components/Settings/FavoritesTab.js

```diff
@@ -10,6 +11,7 @@ export default ({ clubs = [], favorites, keyword, updateFavorites }) => {
     return ret
   })
   const rows = Object.keys(table)
+  const isBookmarks = keyword === 'bookmark'
```

Comment:
Wouldn't correct english grammar be `areBookmarks`? 🤙

Reply:
I've been writing too much code and not enough prose LOL